src/server.ts

```diff
@@ -47,6 +48,7 @@ export function runServer() {
     context.body = getStatus()
   })
   router.post('/setup/address', async context => {
+    await waitForBeeAssetReadiness()
     await createConfigFileAndAddress()
```

Comment:
Actually, it's not super clear to me why is this being done in the `POST /setup/address` endpoint. Shouldn't this be elsewhere and here you just check everything is fine and if not return some code?

Reply:
This is just some UX consideration until it has a standalone step in the installer frontend.

This way the API remains compatible with the installer, this just ensures it waits for the background download in case it is not finished by the time the user gets to the Bee step in the installer flow.